frigate/embeddings/__init__.py

```diff
@@ -28,8 +28,12 @@
 
 
 def manage_embeddings(config: FrigateConfig, metrics: DataProcessorMetrics) -> None:
-    # Only initialize embeddings if semantic search is enabled
-    if not config.semantic_search.enabled:
+    # Only initialize embeddings if semantic search or lpr or face rec is enabled
+    if (
+        not config.semantic_search.enabled
```

Comment:
wouldn't this also work if only genai is enabled?

Reply:
thanks, good catch. I moved this check to app.py, it was redundant here.